packages/core/lib/modules/wallet-backup.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
what's this?

Reply:
should probably move into a tests folder of some sort

Reply:
Yeah, make sense

Reply:
fixed